source/includes/documentation/_vue_js_elements.md

```diff
@@ -2913,55 +2913,67 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
 
 ### 5.29 Images
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/images/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/images/)
 
 Image is located in the following class:
 - __Java__: _package com.epam.jdi.light.vuetify.elements.common.Image.java_
 
 ```java
-    @Test(dataProvider = "measurementImagesDataProvider", dataProviderClass = ImagesTestsDataProvider.class)
-    public void measurementImagesTest(int sliderValue, int width, int height) {
-        aspectRatioImage.is().displayed();
-        slider.slideHorizontalTo(String.valueOf(sliderValue));
-        aspectRatioImage.has().width(width);
-        aspectRatioImage.has().height(height);
+    //@FindBy(css = "#GradientsImage .v-image")
+    @UI("#GradientsImage .v-image")
+    public static Image gradientImage;
+    
+    @Test(description = "Check shown image has gradient")
+    public void gradientsImagesTests() {
+        gradientImage.show();
+        gradientImage.is().displayed()
+            .has().sourcePath("https://cdn.vuetifyjs.com/images/parallax/material2.jpg")
+            .has().width(517)
+            .has().height(291)
+            .has().gradient();
     }
```

Comment:
test should be modified as we discussed on sync

Reply:
Replaced the test